EpamWebSiteTestingFramework/Epam.TestAutomation.Tests/BaseTest.cs

```diff
@@ -0,0 +1,63 @@
+using Epam.TestAutomation.Core.DriverCreator;
+using Epam.TestAutomation.Core.Helper;
+using Epam.TestAutomation.Core.Utils;
+using Epam.TestAutomation.Utilities.Logger;
+using Epam.TestAutomation.Core.ScreenShotMaker;
+using Epam.TestAutomation.Pages.PageObjects.Pages;
+
+namespace Epam.TestAutomation.Tests;
+
+public abstract class BaseTest
+{
+    protected MainPage MainPage { get; set; }
+
+    [OneTimeSetUp]
+    public void OneTimeSetUp()
+    {
+        MyLogger.InitLogger("logs_", UiTestSettings.LogsPath);
+    }
+
+    [SetUp]
+    public void LoggerSetUp()
+    {
+        MyLogger.Info("Test execution is started.");
+    }
+
+    [SetUp]
+    public void DriverSetUp()
+    {
+        Browser.Driver.GotToWebPageUrl(UiTestSettings.ApplicationUrl);
+        Waiter.WaitForPageLoading();
+
+        MainPage = new MainPage();
+    }
+
+    [SetUp]
+    public void PagesSetUp()
+    {
+        MainPage = new MainPage();
+    }
+
+    [TearDown]
+    public void LoggerTearDown()
+    {
+        if (TestContext.CurrentContext.Result.Outcome.Status == NUnit.Framework.Interfaces.TestStatus.Passed)
+        {
+            MyLogger.Info($"Test '{TestContext.CurrentContext.Test.MethodName}' is passed.");
+        }
+
+        if (TestContext.CurrentContext.Result.Outcome.Status == NUnit.Framework.Interfaces.TestStatus.Failed)
```

Comment:
Didn't get

Reply:
add **else** **if** (TestContext.CurrentContext.Result.Outcome.Status == NUnit.Framework.Interfaces.TestStatus.Failed) instead of simple **if** (TestContext.CurrentContext.Result.Outcome.Status == NUnit.Framework.Interfaces.TestStatus.Failed)